apps/api/resources/api/ViewCollection.ttl

```diff
@@ -18,6 +20,10 @@ PREFIX hydra: <http://www.w3.org/ns/hydra/core#>
       hydra:property rdf:type ;
       hydra:object view:View ;
     ] ;
+  query:memberInclude
+    [
+      query:path acl:owner ;
+    ] ;
```

Comment:
This is feature of creta's default collection handler to [eager-load linked resources](https://creta.hypermedia.app/#/knossos/collections?id=eager-loading-linked-resources).

Here, the owner is being added to the response so that the user name can be displayed on the overview page